src/DGtal/io/viewers/Viewer3DFactory.h

```diff
@@ -192,6 +193,42 @@ namespace DGtal
     // StandardDSS6Computer
 
 
+    // naive3DDSSComputer
```

Comment:
@dcoeurjo Not sure about that since I've already used Board3D with it. I think Display3DFactory does the job.